piet-gpu/bin/android.rs

```diff
@@ -54,9 +54,9 @@ fn my_main() -> Result<(), Error> {
                         let width = window.width() as usize;
                         let height = window.height() as usize;
                         let handle = get_handle(window);
-                        let (instance, surface) = Instance::new(Some(&handle), Default::default())?;
-                        gfx_state =
-                            Some(GfxState::new(&instance, surface.as_ref(), width, height)?);
+                        let instance = Instance::new(InstanceFlags::PRESENT)?;
+                        let surface = instance.surface(&handle)?;
```

Comment:
This is missing an unsafe, but otherwise is tested to work on Pixel 4.
```suggestion
                        let surface = unsafe { instance.surface(&handle)? };
```

Reply:
Thanks! I should have caught that at least :)